### PR TITLE
upd(heroic-games-launcher-deb): `2.5.2` -> `2.6.0`

### PR DESCRIPTION
**Pull request by soumyatheman**

### CODE REVIEW

File: packages/heroic-games-launcher-deb/heroic-games-launcher-deb.pacscript [hunk 1]
```diff
@@ -1,8 +1,8 @@
 name="heroic-games-launcher-deb"
 gives="heroic"
-repology=("project: ${gives}")
-version="2.5.2"
+repology=("project: heroic-games-launcher")
```
Comment (wizard-28) [inline packages/heroic-games-launcher-deb/heroic-games-launcher-deb.pacscript:3]: Why did you change the repology project?

Comment (soumyatheman) [reply packages/heroic-games-launcher-deb/heroic-games-launcher-deb.pacscript:3]: This repology gives latest updates
